<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGraphicItemDialog</class>
 <widget class="QDialog" name="NewGraphicItemDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Graphic Item</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Graphic</string>
   </property>
   <widget class="QLabel" name="ComponentPicturelabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>[Picture]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="PastetoolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>24</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Dimensions</string>
   </property>
   <widget class="QLabel" name="Heightlabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>37</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="HeightSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>35</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
    <property name="value">
     <double>60.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Widthlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>38</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="WidthSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>35</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
    <property name="value">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>37</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Graphic Name</string>
   </property>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGraphicItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGraphicItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
